Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine
+
+ENV TERM=xterm LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8
+ENV TIMEZONE US/Pacific
+
+# https://github.com/nodejs/docker-node/issues/588
+# not sure if it's paxctl -cm or paxctl -cM
+RUN apk -U upgrade --available \
+  && apk add \
+    ca-certificates openssl wget file nano tzdata htop git cmake build-base sqlite \
+    paxctl python2 nodejs \
+  && update-ca-certificates \
```

Comment:
I got a warning here, is this something we know about?
```
WARNING: ca-certificates.crt does not contain exactly one certificate or CRL: skipping
```

Reply:
Um, I've no idea. lol. But probably not important.